### PR TITLE
Adding minimal set of energy service indicators for buildings sector

### PR DESCRIPTION
**Pull request by volker-krey**
includes floor space for residential, commercial and combined buildings sector based on NAVIGATE legacy template

### CODE REVIEW

File: definitions/variable/buildings/service.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- Energy Service|Commercial|Floor Space:
+    description: energy service demand for conditioned floor space in commercial buildings
```
Comment (danielhuppmann) [inline definitions/variable/buildings/service.yaml:2]:
```suggestion
    description: Energy service demand for conditioned floor space in commercial buildings
```

File: definitions/variable/buildings/service.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    unit: bn m2
+    tier: "2"
+- Energy Service|Residential and Commercial|Floor Space:
+    description: energy service demand for conditioned floor space in buildings
```
Comment (danielhuppmann) [inline definitions/variable/buildings/service.yaml:6]:
```suggestion
    description: Energy service demand for conditioned floor space in buildings
```

File: definitions/variable/buildings/service.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    unit: bn m2
+    tier: "2"
+- Energy Service|Residential|Floor Space:
+    description: energy service demand for conditioned floor space in residential buildings
```
Comment (danielhuppmann) [inline definitions/variable/buildings/service.yaml:10]:
```suggestion
    description: Energy service demand for conditioned floor space in residential buildings
```

**Review by danielhuppmann — APPROVED** [review]
Looks good to me, thanks.

Going forward, we will probably add many variables that are identical but defined for commercial, residential, and residential-and-commercial buildings, so you could add a tag-list (i.e., a file called tag_residential_commercial.yaml) like
```
- Residential & Commercial:
    - Residential:
         description: residential
    - Commercial:
         description: commercial
    - Residential and Commercial:
         description: all
```

and the service.yaml could be simpler:
```yaml
Energy Service|{Residential & Commercial}|Floor Space:
     description: Energy service demand for conditioned floor space in {Residential & Commercial} buildings
     unit: bn m2
     tier: "2"
```

**Comment by amastrucci** [comment]
Thanks, looks good to me as well. The tag list sounds like a good solution for future versions.

**Review by amastrucci — COMMENTED** [review]
Changes accepted.

**Pull request merged**